tools/test2markdown/test2markdown.sh

```diff
@@ -36,7 +36,7 @@ if [ ! -d "$SW_TEST_OUT_DIR" ]; then
 fi
 
 echo "Generate software test cases ..."
-pyTRLCConverter --source=../../doc/sw-requirements --source=../../doc/sw-test --exclude=../../doc/sw-requirements --source=../../doc/models -o="$SW_TEST_OUT_DIR" --project=test2markdown "$SW_TEST_OUT_FORMAT"
+pyTRLCConverter --source=../../doc/sw-requirements --source=../../doc/sw-test --exclude=../../doc/sw-requirements --source=../../doc/models -o="$SW_TEST_OUT_DIR" --verbose --project=test2markdown "$SW_TEST_OUT_FORMAT"
```

Comment:
req2docx.sh verbose mode is disabled, here its enabled.
I disabled verbose mode in another branch completely, because all of the information disturbed. We should discuss whether we set or clear it.